js/fetch.ts

```diff
@@ -198,12 +198,17 @@ export async function fetch(
 ): Promise<domTypes.Response> {
   const url = input as string;
   log("dispatch FETCH_REQ", url);
+  const method = init && init.method ? (init.method as string) : "GET";
```

Comment:
👍 

I think you may need to implement Request for this, see #191. Note that .headers is an attribute of Request so you'll be able to send that too. (Maybe .body too?)

Reply:
Maybe see also https://github.com/superfly/fly.rs/blob/master/v8env/src/request.ts